chsdi/views/catalog.py

```diff
@@ -1,14 +1,86 @@
 # -*- coding: utf-8 -*-
 
 from pyramid.view import view_config
-from pyramid.httpexceptions import HTTPNotFound
+import networkx as nx
+from itertools import chain
+from pyramid.httpexceptions import HTTPInternalServerError, HTTPNotFound
 
 from chsdi.models.bod import Catalog
 from chsdi.lib.validation import MapNameValidation
 from chsdi.lib.sqlalchemy_customs import remove_accents
 from chsdi.lib.filters import filter_by_geodata_staging
 
 
+# https://github.com/networkx/networkx/blob/master/networkx/readwrite/json_graph/tree.py#L16
```

Comment:
The comment should mention it. Also, why was a custom one needed? Might it be worth to upstream the adaption?